ccm_web/server/src/api/api.interfaces.ts

```diff
@@ -10,3 +10,13 @@ export interface Globals {
     roles: string[]
   }
 }
+
+export interface APIErrorData {
+  statusCode: number
+  message: string
+}
+
+export function isAPIErrorData (v: unknown): v is APIErrorData {
```

Comment:
`v` can have better name like `response` or `httpStatus`

Reply:
I could change it to `value`? I don't want to change it to one of the ones you suggested because we don't yet know what the value is.

Reply:
Ok `value` would be better name than `v`

Reply:
See https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/121/commits/048dd661b3e7e57dea83b320c8bce535374af213